yet-another-docker-plugin/pom.xml

```diff
@@ -91,12 +98,12 @@
                     <dependentWarExcludes>
                         org.apache.httpcomponents:*,com.google.guava*
                     </dependentWarExcludes>
-                    <archive>
-                        <manifestEntries>
-                            <Mask-Classes>com.google.common</Mask-Classes>
-                        </manifestEntries>
-                    </archive>
-                    <maskClasses>com.google.common.</maskClasses>
+                    <!--<archive>-->
```

Comment:
@jglick i commented mask-classes to exclude masking classloader, but it didn't help.